src/bundle/Resources/public/js/scripts/admin.input.text.js

```diff
@@ -8,20 +7,13 @@
         textInput.dispatchEvent(new Event('input'));
         textInput.select();
     };
+    const attachListenersToAllInputs = () => {
+        const textInputClearBtns = doc.querySelectorAll('.ibexa-input-text-wrapper__action-btn--clear');
 
-    textInputClearBtns.forEach((clearBtn) => clearBtn.addEventListener('click', clearText, false));
-
-    doc.body.addEventListener(
-        'ibexa-drop-field-definition',
-        (event) => {
-            const { nodes } = event.detail;
+        textInputClearBtns.forEach((clearBtn) => clearBtn.addEventListener('click', clearText, false));
+    };
 
-            nodes.forEach((node) => {
-                const fieldTextInputClearBtns = node.querySelectorAll('.ibexa-input-text-wrapper__action-btn--clear');
+    doc.body.addEventListener('ibexa-new-inputs-added', attachListenersToAllInputs, false);
```

Comment:
Do you have any ideas for a better event name?

Reply:
`ibexa-inputs:added`?